FastSimulation/MaterialEffects/interface/NuclearInteractionSimulator.h

```diff
@@ -14,6 +14,7 @@
  */
 
 #include "FastSimulation/MaterialEffects/interface/MaterialEffectsSimulator.h"
+#include "DQMServices/Core/interface/DQMStore.h"
```

Comment:
@schneiml @ssekmen @sbein as ME are commented, is this (and the previous update to BuildFile) really needed? Otherwise it would be good to clean it (possibly in a separate PR)